static/app/components/workflowEngine/form/control/priorityControl.spec.tsx

```diff
@@ -4,14 +4,17 @@ import Form from 'sentry/components/forms/form';
 import FormModel from 'sentry/components/forms/model';
 import PriorityControl from 'sentry/components/workflowEngine/form/control/priorityControl';
 import {PriorityLevel} from 'sentry/types/group';
+import {METRIC_DETECTOR_FORM_FIELDS} from 'sentry/views/detectors/components/forms/metricFormData';
 
 describe('PriorityControl', function () {
   it('renders children', async function () {
     const formModel = new FormModel({
       initialData: {
-        'conditionGroup.conditions.0.type': 'above',
-        'conditionGroup.conditions.0.comparison': '0',
-        'conditionGroup.conditions.0.conditionResult': PriorityLevel.LOW,
+        [METRIC_DETECTOR_FORM_FIELDS.initialPriorityLevel]: 'low',
+        [METRIC_DETECTOR_FORM_FIELDS.conditionType]: 'gt',
+        [METRIC_DETECTOR_FORM_FIELDS.conditionValue]: '0',
```

Comment:
what's a `conditionValue`? should this type be consistent on FE / BE?

Reply:
conditionValue is this. I could see calling it thresholdValue 
![image](https://github.com/user-attachments/assets/8777e8ab-d918-4328-a6b6-938ae1bd4b27)